exercises/minesweeper/MinesweeperTest.cs

```diff
@@ -1,235 +1,160 @@
 // This file was auto-generated based on version 1.1.0 of the canonical data.
 
-using Xunit;
 using System;
+using Xunit;
 
 public class MinesweeperTest
 {
     [Fact]
     public void No_rows()
     {
-        var minefield = new string[0];
-        var expected = new string[0];
-        Assert.Equal(expected, Minesweeper.Annotate(minefield));
+        var minefield = Array.Empty<string>();
+        Assert.Empty(Minesweeper.Annotate(minefield));
     }
 
     [Fact(Skip = "Remove to run test")]
     public void No_columns()
     {
-        var minefield = new string[] 
-        { 
-            "" 
-        };
-        var expected = new string[] 
-        { 
-            "" 
+        var minefield = new[]
```

Comment:
Minor: the fact that each row is a separate line here vs in the expected it's on one line makes this definitely look like it wasn't handwritten.

Reply:
Fixed